src/package.xml

```diff
@@ -1,21 +1,33 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Package xmlns="http://soap.sforce.com/2006/04/metadata">
-    <types>
-        <members>Insert_Student_Details</members>
-        <members>StudentQuery</members>
-        <name>ApexClass</name>
-    </types>
```

Comment:
Add custom fields for the relationship Between **Contact** & **Pan Card** and **Lead** & **Account**.